rfd/0058-package-distribution.md

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
Same remark as https://github.com/gravitational/teleport/pull/13197#pullrequestreview-1001925509, we probably don't need to backport the RFD itself.